terraform/ansible/roles/storageclass/tasks/main.yml

```diff
@@ -2,5 +2,7 @@
 - name: use file-gold-gid
   shell: |
     OLD_STORAGE_CLASS=$(KUBECONFIG={{ kube_config }} {{ lookup('env', 'HOME') }}/bin/kubectl get sc -o jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubernetes\.io\/is-default-class=="true")].metadata.name}')
-    KUBECONFIG={{ kube_config }} {{ lookup('env', 'HOME') }}/bin/kubectl patch storageclass ibmc-file-gold-gid -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
-    KUBECONFIG={{ kube_config }} {{ lookup('env', 'HOME') }}/bin/kubectl patch storageclass ${OLD_STORAGE_CLASS} -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
\ No newline at end of file
+    if [ "$OLD_STORAGE_CLASS" != "ibmc-file-gold-gid" ]; then
+      KUBECONFIG={{ kube_config }} {{ lookup('env', 'HOME') }}/ansible/bin/kubectl patch storageclass ibmc-file-gold-gid -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+      KUBECONFIG={{ kube_config }} {{ lookup('env', 'HOME') }}/ansible/bin/kubectl patch storageclass ${OLD_STORAGE_CLASS} -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
```

Comment:
Classic cluster deploy failed at the storage class task, with error "/home/nobody/ansible/bin/kubectl: No such file or directory". I suspect "ansible" should not be in the path?